packages/bootstrap-chat/README.md

```diff
@@ -131,6 +131,10 @@ The arguments for this render prop are the same as for `renderTextResponse`, wit
 
 A callback that will get called any time a new message is received. The callback will receive as an argument all messages received thus far.
 
+### `propAppSettings`
```

Comment:
the approach looks good to me, i would just rename the prop to something like `defaultAppSettings`, using `prop-` in a prop name isn't very conventional